api-report/container-definitions.api.md

```diff
@@ -245,6 +245,10 @@ export type ICriticalContainerError = IErrorBase;
 export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>, IDeltaSender {
     readonly active: boolean;
     readonly clientDetails: IClientDetails;
+    // @deprecated (undocumented)
+    dispose(error?: Error): void;
```

Comment:
this doesn't look right. we don't usually add pre-deprecated things to an interface.  is this a mistake, as it doesn't appear used

Reply:
I didn't change dispose, so I'm not sure why it generated this.